streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignorTest.java

```diff
@@ -809,14 +888,37 @@ public void testAssignWithStates() {
                    APPLICATION_ID + "-store3-changelog"),
             asList(3, 3, 3))
         );
-        configureDefault();
+
+        createDefaultMockTaskManager();
+        final List<Map<String, List<TopicPartitionInfo>>> changelogTopicPartitionInfo = getTopicPartitionInfo(3,
```

Comment:
nit: move first parameter `3,` into it's own line below (hard to read the code otherwise)